data/mods/MindOverMatter/powers/electrokinesis.json

```diff
@@ -531,6 +531,49 @@
     "targeted_monster_species": [ "CYBORG", "ROBOT" ],
     "ignored_monster_species": [ "YRAX_CONSTRUCT", "PSI_NULL" ]
   },
+  {
+    "id": "electrokinetic_robot_interface",
+    "type": "SPELL",
+    "name": "[Ψ]Robotic Interface",
+    "description": "Use your powers to remotely interface with a robot's internal circuitry, allowing you to hack it if you have the skill.",
+    "teachable": false,
+    "valid_targets": [ "self" ],
+    "spell_class": "ELECTROKINETIC",
+    "skill": "metaphysics",
+    "flags": [ "PSIONIC", "CONCENTRATE", "SILENT", "NO_HANDS", "NO_LEGS", "NO_EXPLOSION_SFX" ],
+    "difficulty": 10,
```

Comment:
What Difficulty do you intend the power to be? Here it says Difficulty 10, the contemplation recipe assumes it's Difficulty 9, and internals of the contemplation recipe assume Difficulty 8.

Reply:
Oh, oops.  Sorry--the contemplation recipes are always one below the Difficulty of the power they train, so that they'll raise Metaphysics to the level of the Difficulty and not one above it.

Reply:
Thanks for the clarification on the contemplation difficulty.  Reduced it to 7.

For the 16 difficulty, I wanted to try also adding in the user's computer skill to the roll.  This way, knowing a lot of about how to hack into computers would make it easier to figure out how to use your powers to hack computers, and not knowing how to use computers would make it harder.

Reply:
Oh, right! You mentioned that and I completely missed it when I was reading through the power. That makes sense.